src/main/java/edu/harvard/iq/dataverse/settings/FeatureFlags.java

```diff
@@ -30,7 +30,12 @@ public enum FeatureFlags {
      * @since Dataverse 5.14
      */
     API_SESSION_AUTH("api-session-auth"),
-    
+    /**
+     * Enables API authentication via Bearer Token.
+     * @apiNote Raise flag by setting "dataverse.feature.api-bearer-auth"
+     * @since Dataverse 5.14
```

Comment:
Just a heads up that this may not make it into 5.14 (which we're trying to ship before the community meeting next month). Also, it's a little uncertain if the release after that will be 5.15 or 6.0! 😅